.github/workflows/cassini-predict-spk-regression.yml

```
@@ -0,0 +1,59 @@
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  cassini-predict-spk-regression:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    env:
+      SATELLITE_RF_CASSINI_KERNEL_ROOT: ${{ runner.temp }}/cassini-predict-kernels
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)